juan-pablo-hernandez/tarea-3/oddOrEven.js

```diff
@@ -0,0 +1,10 @@
+function oddOrEven(array) {
```

Comment:
@juan-hm , find a way to make this a one-line solution also use more than one return expression is a bad practice